addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18330,13 +18365,80 @@ msgid "Select emulator for {0:s} file"
 msgstr ""
 
 #. Label on emulators with a savestate in the emulator selection dialog
-#: xbmc/games/dialogs/GUIDialogSelectGameClient.cpp
 #: xbmc/music/windows/GUIWindowMusicPlaylistEditor.cpp
 msgctxt "#35259"
 msgid "Saved"
 msgstr ""
 
-#empty strings from id 35260 to 35504
+#. Title of the dialog to select a savestate for the current game
+#: xbmc/games/dialogs/GUIDialogSelectSavestate.cpp
+msgctxt "#35260"
+msgid "Select savestate"
+msgstr ""
+
+#. Button for creating a new save state from the select savestate dialog
+#: xbmc/games/dialogs/GUIDialogSelectSavestate.cpp
+msgctxt "#35261"
+msgid "New game"
+msgstr ""
+
+#. Help text for RetroAchievements username input setting
+#: system/settings/settings.xml
+msgctxt "#35262"
+msgid "Enter your RetroAchievements account username"
+msgstr ""
+
+#. Help text for RetroAchievements password input setting
+#: system/settings/settings.xml
+msgctxt "#35263"
+msgid "Enter your RetroAchievements account password"
+msgstr ""
+
+#. Label for notification events regarding RetroAchievements login
+#: xbmc/games/GameSettings.cpp
+msgctxt "#35264"
+msgid "RetroAchievements"
+msgstr ""
+
+#. Description for notification event regarding unsuccessful RetroAchievements login
+#: xbmc/games/GameSettings.cpp
+msgctxt "#35265"
+msgid "Incorrect User/Password!"
+msgstr ""
+
+#. Description for notification event regarding unsuccessful RetroAchievements login
+#: xbmc/games/GameSettings.cpp
+msgctxt "#35266"
+msgid "Failed to contact server"
+msgstr ""
+
+#. Description for notification event regarding unsuccessful RetroAchievements login
+#: xbmc/games/GameSettings.cpp
+msgctxt "#35267"
+msgid "Invalid response from server"
+msgstr ""
+
+#. Setting to show if the user is logged in to RetroAchievements
+#. Settings -> Games -> Achievements -> Logged in
+#: system/settings/settings.xml
+msgctxt "#35268"
+msgid "Logged in"
+msgstr ""
+
+#. Help text for the setting that shows if the user is logged in to RetroAchievements
+#. Settings -> Games -> Achievements -> Logged in
+#: system/settings/settings.xml
+msgctxt "#35269"
+msgid "Toggle this setting to log in or out of RetroAchievemts."
```

Comment:
Typo: RetroAchievemts => RetroAchievements

Reply:
I added a fix in https://github.com/xbmc/xbmc/pull/21192